packages/website/gatsby-node.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Implement Gatsby's Node APIs in this file.
+ *
+ * See: https://www.gatsbyjs.org/docs/node-apis/
+ */
+
+const path = require('path')
+const { createFilePath } = require(`gatsby-source-filesystem`)
+
+exports.createPages = async ({ graphql, actions }) => {
+  const { createPage, createRedirect } = actions
+  const postTemplate = path.resolve('./src/templates/doc-page.js')
+  const defaultPath = '/docs/getting-started/'
+
+  const allDocs = await graphql(
+    `
+      {
+        allMdx(limit: 1000) {
+          edges {
+            node {
+              fields {
+                slug
+              }
+            }
+          }
+        }
+      }
+    `,
+  )
+
+  if (allDocs.errors) {
+    console.error(`Error in allDocs: ${allDocs.errors}`)
+    throw Error(`Error in allDocs: ${allDocs.errors}`)
```

Comment:
Doesn't this `throw` get logged automatically? I personally don't know, just would like to make sure we need the `console.error`.